zng/docs/spec.md

```diff
@@ -620,23 +624,23 @@ type descriptor of the line using the formats shown in the
 
 Here are some simple examples to get the gist of the ZNG text format.
 
-Primitive types look like this and do not need typedefs:
+Primitive types look like this:
```

Comment:
Late-breaking change: Now that I've brought @henridf around to the subtleties of invoking TZNG in the "ZNG Data Model" section [😄  ] I'm realizing the TZNG Examples seems to need one last adjustment.

1. "Typedefs" aren't really "a thing" in TZNG
2. While "Type Bindings" _are_ a thing in TZNG, it's not like I could just do a 1-for-1 swap of the terms, since as we show in the examples, you _would_ need to do a Type Binding first if you wanted to express a standalone Value of a primitive type (this being something not even possible in `zq` today, admittedly)

Therefore, I've opted to just remove the wording that references "typedefs" in this section and otherwise leave the examples as-is. Just showing that the primitive types are written out as-is maybe doesn't communicate much, but it still seems like a legit way to build up to when we start referencing them in the example just below that starts building on type aliases, etc.

Reply:
LGTM